catalog.html

```diff
@@ -0,0 +1,308 @@
+                </svg>
+                Discord
+            </a>
+            <a href="#" class="btn btn--icon btn--color">
+                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M14.5 12H14.51M1 3V17C1 18.1046 1.89543 19 3 19H17C18.1046 19 19 18.1046 19 17V7C19 5.89543 18.1046 5 17 5L3 5C1.89543 5 1 4.10457 1 3ZM1 3C1 1.89543 1.89543 1 3 1H15M15 12C15 12.2761 14.7761 12.5 14.5 12.5C14.2239 12.5 14 12.2761 14 12C14 11.7239 14.2239 11.5 14.5 11.5C14.7761 11.5 15 11.7239 15 12Z" stroke="#F5F8FB" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+                Connect
+            </a>
+        </div>
+    </header>
+
+
+    <main class="main main--two-col">
+        <aside class="sidebar">
+            <details class="faq__item">
+                <summary class="faq__question">What are the NFTs?</summary>
+                <p class="faq__answer">Urna vitae erat et lacus, consectetur ac nulla vestibulum lobortis. Nulla dapibus urna volutpat venenatis, risus faucibus</p>
+            </details>
+        </aside>
+
+        <article class="blog-content">
+
```

Comment:
Please add title, select form, search form and buttons list here.

Reply:
done